### PR TITLE
RHINENG-9025 add support to test playbook dispatcher run events in local

### PR DESCRIPTION
**Pull request by r14chandra**
## Why do we need this change? :thought_balloon:

To test our dispatcher-consumer service.

## Documentation update? :memo:

- [X] Yes
- [ ] No

## :guardsman: Checklist :dart:

- [ ] Bugfix
- [ ] New Feature
- [ ] Refactor
- [ ] Unittests Added
- [ ] DRY code
- [ ] Dependency Added
- [ ] DB Migration Added

## Additional :mega:

Feel free to add any other relevant details such as __links, notes, screenshots__, here.

### CODE REVIEW

File: internal/config/config.go [hunk 1]
```diff
@@ -75,7 +75,7 @@ var DefaultConfig Config = Config{
 	AWSRegion:               "us-east-1",
 	AWSSecretAccessKey:      os.Getenv("CW_AWS_SECRET_ACCESS_KEY"),
 	CloudConnectorClientID:  "config-manager",
-	CloudConnectorHost:      flagvar.URL{Value: url.MustParse("http://cloud-connector:8080")},
+	CloudConnectorHost:      flagvar.URL{Value: url.MustParse("http://localhost:8080")},
```
Comment (patilsuraj767) [inline internal/config/config.go:78]: Any specific reason for all these changes in config.go? We are defining this in Makefile - [here](https://github.com/RedHatInsights/config-manager/blob/master/Makefile#L43).

Comment (r14chandra) [reply internal/config/config.go:78]: Why are these added as default instead of localhost?  Where are these hosts defined?

Comment (patilsuraj767) [reply internal/config/config.go:78]: I am not sure, that is the reason I did not changed it initially and instead added flags in the Makefile command.

Comment (r14chandra) [reply internal/config/config.go:78]: Yes, that's why I changed it to `localhost` because accessing API using curl would fail if we use these hosts.

**Review by patilsuraj767 — APPROVED** [review]
LGTM 👍

**Pull request merged**